benchmarker/src/benchmarker.cr

```diff
@@ -77,7 +78,7 @@ end
 # Running client and returning span
 def client
   s = Time.now
-  `#{CLIENT} -t 16 -r 5000`
+  `#{CLIENT} -t 16 -r 256`
```

Comment:
Actually `-r` option doesn't mean the concurrency but number of requests.
I will fix the option by myself, so please remove the fix.
Thanks for the idea.